### ISSUE TITLE
Configuring login_not_required for django_browser_reload with Django 5.1's LoginRequiredMiddleware

### ISSUE THREAD

**Issue by Macktireh** [adamchainz/django-browser-reload#281]
### Python Version

3.12.3

### Django Version

5.1

### Package Version

1.13.0

### Browser

Chrome & Firefox

### Description

With Django 5.1's new `LoginRequiredMiddleware`, how can I set `login_not_required` for django_browser_reload:

```py
path("__reload__/", include("django_browser_reload.urls")),
```

Thanks in advance

**Comment by adamchainz** [comment]
Fixed in #283, released in 1.14.0.

**Comment by Macktireh** [comment]
@adamchainz Thank you very much
